h/groups/templates/create.html.jinja2

```diff
@@ -12,29 +12,7 @@
   <div class="content paper">
     {% include "h:templates/includes/header.html.jinja2" %}
     <div class="form-vertical">
-      <form class="form" action="{{ form.action }}" method="{{ form.method }}" novalidate>
```

Comment:
This is going to conflict with the changes in https://github.com/hypothesis/h/pull/2577 . We'll have to figure out how to reconcile those tomorrow.